### PR TITLE
[api] Add open action to Fulfillment resource

### PR DESCRIPTION
**Pull request by krzysieq**
This adds the [open](https://help.shopify.com/api/reference/fulfillment#open) action to the Fulfillment resource.

### CODE REVIEW

File: resources/fulfillment.js [hunk 1]
```diff
@@ -37,6 +37,20 @@ Fulfillment.prototype.complete = function complete(orderId, id) {
...
+ *
+ * @param {Number} orderId Order ID
+ * @param {Number} id Fulfillment id
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+Fulfillment.prototype.open = function open(orderId, id) {
+  const url = this.buildUrl(orderId, `${id}/open`);
+  return this.shopify.request(url, 'POST', undefined, {})
+      .then(body => body[this.key]);
```
Comment (lpinca) [inline resources/fulfillment.js]: Please fix the indentation here (2 spaces).

File: test/fulfillment.test.js [hunk 2]
```diff
@@ -108,6 +108,17 @@ describe('Shopify#fulfillment', () => {
       .then(data => expect(data).to.deep.equal(output.fulfillment));
   });
 
+  it('opens a pending fulfillment', () => {
+    const output = fixtures.res.open;
+
+    scope
+        .post('/admin/orders/450789469/fulfillments/255858046/open.json', {})
```
Comment (lpinca) [inline test/fulfillment.test.js]: Same here and the next occurrences.

**Comment by lpinca** [comment]
Looks good, only a couple of nits.

**Comment by krzysieq** [comment]
Apologies for that, the indentation issues should be fixed now.

**Comment by lpinca** [comment]
Thank you. I'll cut a new release tomorrow.

**Comment by krzysieq** [comment]
Awesome. Thank you for all your work.

**Pull request merged**